src/vxingest/netcdf_to_cb/netcdf_builder.py

```diff
@@ -639,7 +639,37 @@ def umask_value_transform(self, params_dict):
             )
             return None
 
-    def handle_wind_dir_U(self, params_dict):
+    def handle_rh(self, params_dict):
+        """Retrieves a RH value, if there is one,
+           and if not derives the RH from the dewpoint and temperature.
+        Args:
+            params_dict (dict): named function parameters
+        Returns:
+            float: the RH
+        """
```

Comment:
I may have missed it, but it doesn't appear that this first tries to retrieve the RH value - it calculates it instead.

For my own knowledge, what is `recNum`? The name isn't significant to me. (yet!)

Reply:
good catch. The comment is wrong. I'll change it. There aren't any RH values in the METAR netcdf files. The recnum is the data record number (index) into the netcdf data array.

Reply:
Gotcha. That makes sense then!

And thanks for that context! I'll try to remember that for when I'm next in the codebase.